clinguin_ui/ui_menu_bar.lp

```diff
@@ -0,0 +1,183 @@
+#const help_message = @concat(
+    "<p><i>This UI allows to create a configuration</i></p>",
+    "<h5>Add objects</h5>",
+    "<p>In the add 'Add object' section one can add a new object to the system. Only leaf classes can be added as objects. Click on + to add the </p>",
+    "<h5>Edit objects</h5>",
+    "<p>Click on the node in the configuration graph to edit the object. The lower right pannel will show the selected node and posible changes. </p>",
+    "<h5>Partial and full configuration</h5>",
+    "<p>The yellow button on the menu can toggle between partial and full configuration. In partial configuration objects might not have values for attributes and lowebounds might not be filled. In full configuration the system will enforce the complete configuration. </p>",
+    "<h5>Find incrementally</h5>",
+    "<p>Click on the 'Find incrementally' button to allow the system to create all the required objects to complete de configuration. </p>",
+    "<h5>Next solution</h5>",
+    "<p>Browse the solutions for a complete configuration. The new information is highlighted in green.</p>",
+    "<h5>Select a solution</h5>",
+    "<p>When browsing the solutions, select the one that you want to keep. </p>",
+    ""
+).
+% ================= Menu bar
+
+elem(menu_bar,menu_bar,w).
+attr(menu_bar,title,"OOASP").
+attr(menu_bar,icon,"fa-puzzle-piece").
+
+    % ---------- Help ----------
+    elem(help_btn,button,menu_bar).
+    attr(help_btn,icon,"fa-circle-question").
+    attr(help_btn,class,("btn-light")).
+    when(help_btn,click,update,(help_modal,visibility,shown)).
+
+        elem(help_modal,modal,w).
+        attr(help_modal,title,"Help").
+        attr(help_modal,size,"xl").
+        attr(help_modal,class,("justify-content-start")). %Check if this actually does anything.
```

Comment:
What does this comment mean?

Reply:
It was just a note for myself that I forgot to remove.
Gone now.